assets/predictive-search.js

```diff
@@ -263,6 +289,10 @@ class PredictiveSearch extends SearchForm {
     this.resultsMaxHeight = false
     this.predictiveSearchResults.removeAttribute('style');
   }
+
+  clearResources() {
```

Comment:
Can you use the existing `closeResults` method for this instead? Is this additional method necessary?